makefile

```diff
@@ -6,7 +6,7 @@ nodeDir := public
 packages := $(name) agent operations cloud command db util plugin units
 packages += thirdparty auth scheduler model validator service monitor repotracker
 packages += model-patch model-artifact model-host model-build model-event model-task model-user model-distro model-manifest model-testresult
-packages += model-grid rest-client rest-data rest-route rest-model migrations trigger model-alertrecord model-notification model-stats
+packages += model-grid model-role rest-client rest-data rest-route rest-model migrations trigger model-alertrecord model-notification model-stats
```

Comment:
if we were going to do this (and I don't think we should, we also need to add it to the evergreen config file.)

Reply:
moved to the user package